stable/appmesh-gateway/templates/psp.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if .Values.rbac.pspEnabled }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ template "appmesh-gateway.fullname" . }}
+  labels:
+{{ include "appmesh-gateway.labels" . | indent 4 }}
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
+spec:
+  privileged: false
+  hostIPC: false
+  hostNetwork: false
+  hostPID: false
+  readOnlyRootFilesystem: false
+  allowPrivilegeEscalation: false
+  allowedCapabilities:
+    - '*'
```

Comment:
Just to confirm, this will enable the pod with `CAP_NET_BIND_SERVICE`? This is a [known edge](https://docs.aws.amazon.com/app-mesh/latest/userguide/troubleshooting-setup.html#virtual-gateway-low-ports) that's unique about gateway, given we bind to the specified port instead of 15000 for ingress.

Reply:
Yes that should be the case.